From 63a34e9f0906ef0507933a5ffa0ecfb9163d11d9 Mon Sep 17 00:00:00 2001
From: Guanhua Gao <guanhua.gao@nxp.com>
Date: Thu, 10 May 2018 17:55:04 +0800
Subject: [PATCH 571/706] crypto: caam: Fix header file of mc-bus

Fix header file of mc-bus.

Signed-off-by: Guanhua Gao <guanhua.gao@nxp.com>
(cherry picked from commit 848c24134ae3df5b81d9a4abec0265ac6e5dfa75)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/crypto/caam/caamalg_qi2.c | 2 +-
 drivers/crypto/caam/dpseci.c      | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/crypto/caam/caamalg_qi2.c b/drivers/crypto/caam/caamalg_qi2.c
index 49b553746eee..454ad667ebc9 100644
--- a/drivers/crypto/caam/caamalg_qi2.c
+++ b/drivers/crypto/caam/caamalg_qi2.c
@@ -32,6 +32,7 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include <linux/fsl/mc.h>
 #include "compat.h"
 #include "regs.h"
 #include "caamalg_qi2.h"
@@ -43,7 +44,6 @@
 #include "key_gen.h"
 #include "caamalg_desc.h"
 #include "caamhash_desc.h"
-#include "../../../drivers/staging/fsl-mc/include/mc.h"
 #include "../../../drivers/staging/fsl-mc/include/dpaa2-io.h"
 #include "../../../drivers/staging/fsl-mc/include/dpaa2-fd.h"
 
diff --git a/drivers/crypto/caam/dpseci.c b/drivers/crypto/caam/dpseci.c
index dec05ecbeab1..334d8a9cd02e 100644
--- a/drivers/crypto/caam/dpseci.c
+++ b/drivers/crypto/caam/dpseci.c
@@ -32,7 +32,7 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-#include "../../../drivers/staging/fsl-mc/include/mc.h"
+#include <linux/fsl/mc.h>
 #include "../../../drivers/staging/fsl-mc/include/dpopr.h"
 #include "dpseci.h"
 #include "dpseci_cmd.h"
-- 
2.17.1

